io/src/openni_camera/openni_driver.cpp

```diff
@@ -190,12 +190,12 @@ openni_wrapper::OpenNIDriver::updateDeviceList ()
 
 
   // redundant, but needed for Windows right now and also for Xtion
-  for (size_t deviceIdx = 0; deviceIdx < device_context_.size (); ++deviceIdx)
+  for (const auto &deviceIdx : device_context_)
```

Comment:
Using `deviceIdx` for this variable a little misleading. Shouldn't it be just `device` for instance?